storm-core/src/jvm/backtype/storm/tuple/Tuple.java

```diff
@@ -37,9 +37,15 @@
     /**
      * Returns the global stream id (component + stream) of this tuple.
```

Comment:
@mjsax 
Could you append comment to explain 'broken naming convention, use getSourceGlobalStreamId() instead'?